processing/src/main/java/org/apache/carbondata/processing/newflow/sort/unsafe/UnsafeSortDataRows.java

```diff
@@ -306,7 +309,9 @@ private void writeData(UnsafeCarbonRowPage rowPage, File file)
    * This method will be used to delete sort temp location is it is exites
    */
   public void deleteSortLocationIfExists() {
-    CarbonDataProcessorUtil.deleteSortLocationIfExists(parameters.getTempFileLocation());
+    for (String loc : parameters.getTempFileLocation()) {
```

Comment:
could have changed the method signature of CarbonDataProcessorUtil.deleteSortLocationIfExists to take list